backend/app/testbot/main.py

```
@@ -0,0 +1,47 @@
+    ]
+    if not knowledge_base_exists(chatbot_A_id):
+        print(f"Building knowledge base for {chatbot_A_id}...")
+        get_or_create_kb(chatbot_A_id, data_paths_A)
+    else:
+        print(f"Knowledge base for {chatbot_A_id} already exists.")
+    
+    # Build KB for Chatbot B
+    chatbot_B_id = "chatbot_B"
+    dir_chatbot_B = os.path.join(data_dir, "chatbot_B")
+    data_paths_B = [
+        os.path.join(dir_chatbot_B, "products.xlsx")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)